WooCommerce/Classes/ViewRelated/ReusableViews/LabelAndButtonTableViewCell.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+class LabelAndButtonTableViewCell: UITableViewCell {
```

Comment:
This one could be final too

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4910/commits/8c373612c6e33caf4fd8227777a42d7959f0d41e